tests/autobahn/test_autobahn.py

```diff
@@ -0,0 +1,32 @@
+import os
+import shutil
+
+import pytest
+from python_on_whales import DockerClient
+
+
+@pytest.fixture(scope="session", autouse=True)
+def create_report_directory():
+    path = "reports"
```

Comment:
This would probably have to be gitignored. Maybe it's best to have a `.test-results/` dir in the repo with everything gitignored as a designated location for various reports.

Reply:
The `report` directory is added to .gitignore in the tests/autobahn directory, should I remove it and create a `test-results` instead?